airflow/include/tasks/extract/github.py

```diff
@@ -203,7 +204,7 @@ def extract_github_issues(repo_base: str, github_conn_id: str) -> pd.DataFrame:
     gh_hook = GithubHook(github_conn_id)
 
     repo = gh_hook.client.get_repo(repo_base)
-    issues = repo.get_issues(state="all")
+    issues = repo.get_issues(state="all", since=datetime(2022, 1, 1))
```

Comment:
This date should be environment variable or something that can be configured from UI

Reply:
updated PTAL